ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx

```diff
@@ -229,3 +231,26 @@ export function combineSearchFilterWithScanConfig(
         ...createScanConfigFilter(selectedScanConfigName),
     };
 }
+
+export function getScanConfigurationSelectData(
```

Comment:
Here is an alternative to compare and contrast:
1. Higher order util function returns an `isDisabledScanConfiguration` predicate function for individual scan configuration (the guts of the business logic here) instead of `map` to a derived data type.
2. `ScanConfigurationSelect` component has
    * `isDisabledScanConfig: (scanConfig: ComplianceScanConfigurationStatus) => boolean` prop
    * `scanConfigs: ComplianceScanConfigurationStatus[]` prop with same type as elsewhere
3. Rendering components compute predicate function instead of derived data array.